app/controllers/auctions_controller.rb

```diff
@@ -1,13 +1,13 @@
 class AuctionsController < ApplicationController
   def index
     collection = AuctionQuery.new.public_index
-    @view_model = AuctionsIndexViewModel.new(current_user, collection)
+    @auctions = AuctionsIndexViewModel.new(auctions: collection, current_user: current_user)
```

Comment:
I guess we could make the theoretical case that you could see what another user sees even if the current_user is different (like an admin preview), but that is a bit of a stretch. I think the reason I like renaming it though is to distinguish between a `user` variable and a call to the `current_user` method in a controller